addons/website_hr_recruitment/static/tests/tours/website_hr_recruitment.js

```diff
@@ -3,38 +3,114 @@ odoo.define('website_hr_recruitment.tour', function(require) {
 
     var tour = require("web_tour.tour");
 
-    tour.register('website_hr_recruitment_tour', {
```

Comment:
Not entirely convinced we should rename this tour in stable. Technically, you could rely on it. But I am not entirely sure to understand your double tour with double application. You made an utility method to apply for a job, why not calling it twice in a unique tour with the same name?  Launching two tours is way slower, you need to spawn a chrome headless process etc. The only times we call `start_tour` multiple times is when there a need for some python code between the two calls or that you have to switch the user which runs the tour.